src/plugins/data_source_management/README.md

```diff
@@ -0,0 +1,11 @@
+# dataSourceManagement
+
+A OpenSearch Dashboards plugin
```

Comment:
Create an issue to add detail about dsm plugin later

Reply:
https://github.com/opensearch-project/OpenSearch-Dashboards/issues/2031